website/docs/d/client_config.html.markdown

```diff
@@ -29,13 +29,14 @@ There are no arguments available for this data source.
 * `client_id` is set to the Azure Client ID (Application Object ID).
 * `tenant_id` is set to the Azure Tenant ID.
 * `subscription_id` is set to the Azure Subscription ID.
+* `object_id` is set to the Azure Object ID.
 
 ---
 
-~> **Note:** the following fields are only available when authenticating via a Service Principal (as opposed to using the Azure CLI):
+~> **Note:** the following fields are only available when authenticating via a Service Principal (as opposed to using the Azure CLI) and have been deprecated:
```

Comment:
We should probably add these deprecated fields to the Upgrade Guide for 2.0 https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/guides/2.0-upgrade-guide.html.markdown if we haven't already